README.md

```diff
@@ -1,6 +1,6 @@
 # Multiserver Address (Rust)
 
-This is a Multiserver address parser written by Piet Guerson and shared with
+This is a Multiserver address parser written by Piet Geursen and shared with
```

Comment:
:scream:

Reply:
Lol, damn Dutch spelling. You're not the first or the last to fall victim to that.